### PR TITLE
Migration/tickable

### PR DESCRIPTION
**Pull request by rvilarl**
This one is a very straight forward one. :)

### CODE REVIEW

File: src/tickable.ts [hunk 1]
```diff
@@ -8,8 +8,52 @@ import { Vex } from './vex';
...
+  right: number;
+}
+export interface Space {
+  used: number;
+  mean: number;
+  deviation: number;
+}
+export interface FormatterMetrics {
+  duration: string;
+  freedom: LeftRight;
```
Comment (0xfe) [inline src/tickable.ts:28]: Do we need a new type name `LeftRight` here? Can we type it inline?

File: src/tickable.ts [hunk 2]
```diff
@@ -8,8 +8,52 @@ import { Vex } from './vex';
...
+import { ModifierContext } from './modifiercontext';
+import { Tuplet } from './tuplet';
+import { Voice } from './voice';
+import { Modifier } from './modifier';
+
+export interface LeftRight {
+  left: number;
+  right: number;
+}
+export interface Space {
```
Comment (0xfe) [inline src/tickable.ts:21]: How about `Spacing`?

**Comment by 0xfe** [comment]
Looks good overall. There are some unmodified files with lint warnings (for stuff that we're okay with), can you add `eslint-disable-next-line` comments around them in this PR please?

File: src/tickable.ts (hunk 1 above)
Comment (rvilarl) [reply src/tickable.ts:28]: Not really, I change it.

File: src/tickable.ts (hunk 2 above)
Comment (rvilarl) [reply src/tickable.ts:21]: I would leave it as Space. If you search in the code, you will find many references to `space` but none to `spacing`. Your choice anyway.

File: src/tickable.ts (hunk 1 above)
Comment (rvilarl) [reply src/tickable.ts:28]: Now inline.

**Comment by rvilarl** [comment]
> Looks good overall. There are some unmodified files with lint warnings (for stuff that we're okay with), can you add `eslint-disable-next-line` comments around them in this PR please?

@0xfe I would suggest to leave the `warnings` visivisble until the end of the migration. Perhaps we get ideas on the way on how to resolve these `any`

**Comment by 0xfe** [comment]
> > Looks good overall. There are some unmodified files with lint warnings (for stuff that we're okay with), can you add `eslint-disable-next-line` comments around them in this PR please?
> 
> @0xfe I would suggest to leave the `warnings` visivisble until the end of the migration. Perhaps we get ideas on the way on how to resolve these `any`

The warnings start to collect and pollute the PRs, and reviews get messier over item. But worse, they start to mask real warnings that we might want to pay attention to.

I see your point, and I think an alternate strategy is `grep eslint-disable-next-line *.ts`, or grep `TODO *.ts`.and reviews. (It's how I typically handle these types of issues.)

Alternatively, we can create a meta issue around `any` and collect instances that we can review at the end.

File: src/tickable.ts (hunk 2 above)
Comment (0xfe) [reply src/tickable.ts:18]: Okay, it's fine.

**Comment by rvilarl** [comment]
Ready!

**Comment by 0xfe** [comment]
Great, thanks!

**Pull request merged**